### PR TITLE
Fix/1897 select all checkbox

### PR DESCRIPTION
**Pull request by aishmitk**

### CODE REVIEW

File: src/packages/@ncigdc/tableModels/utils.js [hunk 1]
```diff
@@ -109,9 +109,9 @@ export const createSelectColumn: TCreateSelectColumn = ({
     downloadable: false,
     hidden: false,
     th: ({ nodes, selectedIds, setSelectedIds }) => {
-      const ids = nodes.map(node => node[idField]);
+      //TODO: make it easier to understand the structure of nodes obj
```
Comment (alex-wilmer) [inline src/packages/@ncigdc/tableModels/utils.js]: maybe instead:

`// NOTE: "nodes" is really "edges" in the graphql schema`

File: src/packages/@ncigdc/tableModels/utils.js [hunk 2]
```diff
@@ -109,9 +109,10 @@ export const createSelectColumn: TCreateSelectColumn = ({
     downloadable: false,
     hidden: false,
     th: ({ nodes, selectedIds, setSelectedIds }) => {
-      const ids = nodes.map(node => node[idField]);
+      // NOTE: "nodes" is really "edges" in the graphql schema
```
Comment (jbarno) [inline src/packages/@ncigdc/tableModels/utils.js]: Feels like a case to use FlowTypes no? -Not sure if that is possible

File: src/packages/@ncigdc/tableModels/utils.js [hunk 3]
```diff
@@ -109,9 +109,10 @@ export const createSelectColumn: TCreateSelectColumn = ({
...
+      console.log(nodes);
```
Comment (jbarno) [inline src/packages/@ncigdc/tableModels/utils.js]: Debugging log?

File: src/packages/@ncigdc/tableModels/utils.js (hunk 1 above)
Comment (jbarno) [reply src/packages/@ncigdc/tableModels/utils.js]: Or reaname it edges?

File: src/packages/@ncigdc/tableModels/utils.js (hunk 3 above)
Comment (aishmitk) [reply src/packages/@ncigdc/tableModels/utils.js]: ah my bad

**Pull request merged**